src/renderer/components/settings/Information.tsx

```diff
@@ -34,7 +34,7 @@ interface States {
     placeholder: any;
 }
 
-export class LanguageSettings extends React.Component<Props, States> {
+export class AppInformation extends React.Component<Props, States> {
     private parsedMarkdown: string;
```

Comment:
Note that this language-related typo is fixed in this PR, but we could also externalize into a separate PR.